src/snyk/snykCode/falsePositive/falsePositive.ts

```diff
@@ -1,38 +1,63 @@
 import { Marker } from '@snyk/code-client';
 import path from 'path';
 import { IVSCodeWorkspace } from '../../common/vscode/workspace';
+import { completeFileSuggestionType } from '../interfaces';
+import { IssueUtils } from '../utils/issueUtils';
 
 export class FalsePositive {
-  readonly files: Set<string>;
+  private files: Set<string>;
 
-  constructor(private workspace: IVSCodeWorkspace, suggestionFilePath: string, markers: Marker[]) {
+  readonly message: string;
+  readonly id: string;
+  readonly startLine: number;
+  readonly endLine: number;
+  readonly primaryFilePath: string;
+  readonly rule: string;
+  content: string | undefined;
+
+  constructor(private workspace: IVSCodeWorkspace, suggestion: completeFileSuggestionType) {
+    if (!suggestion.markers || suggestion.markers.length === 0) {
```

Comment:
@michelkaporin Why do we link to markers here? Every `suggestion` linked to at least one file. And there are suggestions without `markers` at all but linked to the file. Imho markers is provider of additional connected files but the "primary" one is that file suggestion is directly linked to.

Reply:
@ArtsiomCh hmm, not in my experience when I've debugged it. Markers always had primary file path afair. Can you provide an example where that's not the case? Perhaps something in `nodejs-goof` that I can check?

Reply:
Great catch, synced offline ✔️ To be addressed in ROAD-875.